02-express-tutorial/15-router-controller.js

```diff
@@ -0,0 +1,66 @@
+let { people } = require('../data')
+
+const getPeople = (req, res) => {
+  res.status(200).json({ success: true, data: people })
+}
+
+const createPerson = (req, res) => {
+  const { name } = req.body
+  if (!name) {
+    return res
+      .status(400)
+      .json({ success: false, msg: 'please provide name value' })
+  }
+  res.status(201).send({ success: true, person: name })
+}
+
+const createPersonPostman = (req, res) => {
+  const { name } = req.body
+  if (!name) {
+    return res
+      .status(400)
+      .json({ success: false, msg: 'please provide name value' })
+  }
+  res.status(201).send({ success: true, data: [...people, name] })
+}
+
```

Comment:
Probably not a good idea for security reason to send the all "...people" info to front end, unless for knowledge purpose in this project. For Data integrity! Only send minimal data information to users.